File: python/7 kyu/283.py
# https://www.codewars.com/kata/566fc12495810954b1000030/train/python

# Passed

def nb_dig(n, d):
    squares = [num ** 2 for num in range(n + 1)]
    result = sum(str(square).count(str(d)) for square in squares)
    return result

output = nb_dig(11011, 2)
print(output)

File: nodejs/7 kyu/283.js
// https://www.codewars.com/kata/57b71a89b69bfc92c7000170/train/javascript

// Passed

function getNumberOfSquares(n) {
  let currBase = 0;
  let totalSum = 0;
  while (totalSum < n) {
    currBase++;
    totalSum += currBase ** 2;
  }

  currBase--;
  return currBase;
}

const output = getNumberOfSquares(15);
console.log(output);